src/main/java/org/hydev/mcpm/client/uninstallUseCase/Uninstaller.java

```diff
@@ -0,0 +1,36 @@
+package org.hydev.mcpm.client.uninstallUseCase;
+import java.io.File;
+import java.util.Arrays;
+import org.hydev.mcpm.client.injector.PluginLoader;
+import org.hydev.mcpm.client.injector.PluginNotFoundException;
+
+
+public class Uninstaller implements UninstallBoundary{
+    private boolean remove_depen;
+
+
+    public void uninstall(String name) throws PluginNotFoundException {
+        PluginLoader pil = new PluginLoader();
+        pil.unloadPlugin(name);
+
+        // Find plugin file
+        // could be useful for removing dependencies (?)
+        var plugins_folder = new File("plugins");
```

Comment:
This information should be static. There is a default folder containing the plugins at the relative location "plugins", and there might already be a method that does this work.